src/completionItemBuilder.ts

```diff
@@ -95,12 +97,16 @@ export class CompletionItemBuilder {
   private replaceExpression = (replacement: string, code: string, customRegex?: string) => {
     const re = new RegExp(customRegex || RegexExpression, 'g')
 
-    return replacement.replace(re, (_match, p1) => {
+    let replacedText = replacement.replace(re, (_match, p1) => {
       if (p1 && this.filters[p1]) {
         return this.filters[p1](code)
       }
       return code;
     })
+    const { line } = this.node.getSourceFile().getLineAndCharacterOfPosition(this.node.getStart())
+    const indentStart = vsc.window.activeTextEditor.document.lineAt(line).text.match(/^\s*/)?.[0]
```

Comment:
Yyea I forgot to refactor this, just wanted to check it works & didn't want to add additional arg to all templates.

Reply:
I think it should be passed from [getIndentSize](https://github.com/ipatalas/vscode-postfix-ts/blob/develop/src/postfixCompletionProvider.ts#L122) somehow.
By the way its always safe to put `\n` into snippet as editor always does the normalization.